tenable/io/v3/vm/tags/schema.py

```diff
@@ -0,0 +1,145 @@
+'''
+Tags API Endpoint Schemas
+'''
+from marshmallow import Schema, fields, validate
+from marshmallow.decorators import post_dump, pre_load, validates_schema
+from marshmallow.exceptions import ValidationError
+
+from tenable.io.v3.base.schema.explore.filters import ParseFilterSchema
+
+
+class TagCategorySchema(Schema):
+    '''
+    Schema for tags category methods
+    '''
+    name = fields.Str(required=True)
+    description = fields.Str()
+
+    @validates_schema
+    def validate_name(self, data, **kwargs):
+        '''
+        Supproting method to validate name string length
+        '''
+        if len(data['name']) >= 127:
+            raise ValidationError('The name must not exceed 127 characters.')
```

Comment:
can we use any marshmallow validator ex= v.length(max=127) => pls check the marshmallow document https://marshmallow.readthedocs.io/en/stable/marshmallow.validate.html

Reply:
Resolved